Cython/Utility/ImportExport.c

```diff
@@ -521,37 +510,39 @@ static int __Pyx_SetPackagePathFromImportLib(PyObject *module_name) {
 
 
 /////////////// TypeImport.proto ///////////////
+//@substitute: naming
 
-#ifndef __PYX_HAVE_RT_ImportType_proto
-#define __PYX_HAVE_RT_ImportType_proto
+#ifndef __PYX_HAVE_RT_ImportType_proto_$cy_version
```

Comment:
Wouldn't it much nicer to use Tempita for this?

In the utility code you write
```
__PYX_HAVE_RT_ImportType_proto_{{HASH}}
```

and then you just load it with
```
TempitaUtilityCode.load_cached("TypeImport", "ImportExport.c", context={'HASH': cy_version})
```

Reply:
Disregard my comment. Your approach works just fine, it is easier, and it is used everywhere.